api/base/serializers.py

```diff
@@ -316,7 +402,8 @@ def to_representation(self, obj, envelope='data'):
             except SkipField:
                 continue
 
-            if isinstance(field, JSONAPIHyperlinkedIdentityField):
+            if (isinstance(field, JSONAPIHyperlinkedIdentityField) or isinstance(field, JSONAPIHyperlinkedRelatedField)
```

Comment:
`isinstance` accepts a tuple of possible classes, and will return true so long as the object is an instance of any one of the options.

So for now it would be cleaner to write this as:
`isinstance(obj, (A,B,C))`

Separately: If we expect there to be a family of JSON hyperlinked fields with shared behavior/requirements, it might be a good idea to add a common parent class, so that this could handle any other field types that might be added to that family in the future.

But that level of refactor would be outside the scope of this PR. For now, `isinstance(obj, tuple_of_choices)` will do. :earth_africa: